Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -259,9 +259,15 @@ MainWindow::MainWindow(const QStringList &keywords, bool verbose, QWidget* paren
           SIGNAL(selectionChanged ( const QItemSelection & , const QItemSelection & ) ),
           this, SLOT(selectionChanged()));
   // setup menu filtering
+  #ifndef CGAL_CXX20
   connect(sceneView->selectionModel(),
           QOverload<const QItemSelection & , const QItemSelection &>::of(&QItemSelectionModel::selectionChanged),
           this, [=](){filterOperations(false);});
+  #else
+  connect(sceneView->selectionModel(),
+          QOverload<const QItemSelection & , const QItemSelection &>::of(&QItemSelectionModel::selectionChanged),
+          this, [=,this](){filterOperations(false);});
```

Comment:
why not only capture `[this]` without using the macro defs?

Reply:
@theartful is right: here only the explicit capture of `this` is required. That would simplify the patch.

Reply:
@theartful Ya you are right, I am fixing it, I didn't see that the lambda didn't have anything that needs [=]! My bad! I am leaving the #define CGAL_CXX20 for any future use